src/docs/Intro.stories.mdx

```diff
@@ -0,0 +1,16 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="About/Introduction" />
+
+# Palmetto Design System
+
+Welcome to the Palmetto Design System, whose purpose is to help us deliver high quality Palmetto branded applications, faster and better. It consists of the following following:
+
+- [Palmetto Components](https://github.com/palmetto/palmetto-components) - on open source, react-based, component library for Palmetto applications and websites.
+- [Palmetto Design Tokens](https://github.com/palmetto/palmetto-design-tokens) - A central location to store shared attributes of the Palmetto Design System. These include things like color, fonts, spacing, and more.
+- Brand Assets: _coming soon_
+- Voice and Tone: _coming soon_
+
+If you have any questions or concerns, please feel to reach out to the UX Team on Slack (#ux). One of our missions is build tools to make your job easier and more succesful, so please don't hesitate to reach out.
```

Comment:
"please feel free to"

Reply:
Wording suggestion: "make your job easier and you more successful"?

Reply:
"succesful" is misspelled. The following VSCode plugin is fantastic at catching these things: https://github.com/streetsidesoftware/vscode-spell-checker.

Er, actually, it doesn't work for `.mdx` files, but works great for most other formats.

Reply:
On a more foundational level. Who is the audience for this repo? In the README, we are speaking to developers around the world, hoping that they will use it, and contribute, which is why it's open source. 

However, this line assumes that the audience is made-up of internal Palmetto employees.

Not a pressing matter, but more something to be pondered going forward.

Reply:
most definitely Palmetto employees. a very distant second would be for engineers we are trying to recruit.